bindings/python/pymongoarrow/api.py

```diff
@@ -58,7 +58,7 @@
 _MAX_WRITE_BATCH_SIZE = max(100000, MAX_WRITE_BATCH_SIZE)
 
 
-def find_arrow_all(collection, query, *, schema, **kwargs):
+def find_arrow_all(collection, query, *, schema=None, **kwargs):
     """Method that returns the results of a find query as a
```

Comment:
For all the changed functions, let's update the docs to say that the `schema` param is optional.